examples/applications/Face&EyesDetection.py

```diff
@@ -0,0 +1,42 @@
+'''
```

Comment:
In Arduino library, `examples` directory is intended for Arduino examples.
This is not an Arduino example, so it should go into `extras` directory, which is ignored by Arduino IDE.
See https://arduino.github.io/arduino-cli/0.19/library-specification/

Reply:
Delete this file. Don't add two copies of the same file.